detection-rules/impersonation_benefits_enrollment.yml

```
@@ -0,0 +1,116 @@
+name: "Benefits Enrollment Impersonation"
+description: "Detects messages about benefit enrollment periods and healthcare selections from external senders that contain urgent language or requests for action. Excludes legitimate HR communications, marketing mailers, and trusted sender domains with valid authentication."
+type: "rule"
+severity: "high"
+source: |
+  type.inbound
+  and sender.email.domain.domain not in $org_domains
+  and length(body.current_thread.text) < 2500
+  and (
+    regex.icontains(subject.subject,
+                    '(open|benefits?) enrol{1,2}ment', // catches both enrolment and enrollment
+                    'benefit(s)? (plan|choice|selection|deadline|period)',
+                    'hr benefits',
+                    'annual enrol{1,2}ment',
+                    'healthcare (choice|selection|opt.?in)',
+                    '(fsa|hsa|401k) (enrol{1,2}ment|selection)',
+                    'dependent (coverage|verification)',
+                    '(health|dental|vision|insurance|medical) enrol{1,2}ment'
+    )
+    or regex.icontains(body.current_thread.text,
+                       'benefit(s)? (plan|choice|selection|deadline|period)',
+                       'hr benefits',
+                       'annual enrol{1,2}ment',
+                       'healthcare (choice|selection|opt.?in)',
+                       '(fsa|hsa|401k) (enrol{1,2}ment|selection)',
+                       'dependent (coverage|verification)',
+                       '(health|dental|vision|insurance|medical) enrol{1,2}ment',
+                       '(urgent|immediate) action required.{0,20}(benefit|enrol{1,2}ment)',
+                       'coverage.{0,20}(expire|terminate)',
+                       'last (day|chance).{0,20}(enrol{1,2}|select)',
+                       '(login|sign.?in).{0,20}(benefit portal|hr portal)',
+                       '(verify|update|confirm).{0,20}(benefit.{0,20}selection)'
+    )
+  )
+  and 2 of (
+    any(ml.nlu_classifier(body.current_thread.text).entities,
+        .name in ("urgency", "request")
+    ),
+    any(ml.nlu_classifier(body.current_thread.text).intents, .name != "benign"),
+    (length(body.current_thread.text) < 250 and length(attachments) == 1)
+  )
```

```suggestion
    // lure in attachment
    (
      any(attachments,
          (.file_type in $file_types_images or .file_type in ("pdf", "docx", "doc"))
          and any(filter(file.explode(.), .scan.ocr.raw is not null),
                  (
                    any(ml.nlu_classifier(.scan.ocr.raw).intents,
                        .name != "benign"
                    )
                    or any(ml.nlu_classifier(.scan.ocr.raw).entities,
                            .name in ("urgency", "request")
                    )
                  )
              )
          )
      )
  )
```
detect lures in attachment